backend/attendance-service/src/main/java/org/egov/web/controllers/AttendanceApiController.java

```diff
@@ -2,15 +2,15 @@
 
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import digit.models.coremodels.RequestInfoWrapper;
 import io.swagger.annotations.ApiParam;
 import org.egov.common.contract.response.ResponseInfo;
-import digit.models.coremodels.RequestInfoWrapper;
 import org.egov.service.AttendanceRegisterService;
 import org.egov.util.ResponseInfoFactory;
 import org.egov.web.models.AttendanceRegister;
 import org.egov.web.models.AttendanceRegisterRequest;
 import org.egov.web.models.AttendanceRegisterResponse;
-import org.egov.web.models.AttendanceRegisterSearchCriteria;
+import org.egov.models.AttendanceRegisterSearchCriteria;
```

Comment:
This can stay in web/models. Tgere is a register search api.

Reply:
AttendanceRegisterSearchCriteria file moved to web/models